web/src/components/core/Attribute.jsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) [2023] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+import { Label } from "@patternfly/react-core";
+
+import { Description } from "~/components/core";
+import { Icon } from "~/components/layout";
+
+/**
+ * Display a label with additional details. The details are displayed after
+ * clicking the label and the "i" icon indicates available details.
+ * If the label is not defined or is empty it behaves  like a plain label.
+ * @component
+ *
+ * @param {object} description details displayed after clicking the label
+ * @param {object} children the content of the label
+ */
+export default function Attribute ({ description, children }) {
```

Comment:
NP: I do not like the `Attribute` name. Sounds weird. That said, I have no better suggestion, although the word "badge" comes quickly to my mind.

Reply:
>  Display a label with additional **details**. The **details** are displayed after
>  clicking the label and the "i" icon indicates available **details**.

Maybe the component description is telling us something... `Deatils` could be the name for this component. However, yesterday we somehow agree on avoiding to use names that _collides_ with native HTML elements. And, as you already know because we use it a lot in the PRs descriptions, there is a [`details`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details) HTML element

Reply:
Yeah, the naming is usually the biggest problem... It's not the best name but I could not find any better. 

Suggesstions? Ideas? :thinking:  Anyone...

Reply:
I am aware that I am bikeshedding... but "attribute" sounds like a non-UI thing to me, colliding with an HTML/XML concept.

What about Hint, Tooltip, Tip?

Reply:
Unfortunately "Hint" and "Tooltip" are already used by PatternFly, I'd avoid using the same names.

"Tip" sounds OK.